snmp/datadog_checks/snmp/data/profiles/_cisco-generic.yaml

```diff
@@ -195,3 +195,47 @@ metrics:
           OID: 1.3.6.1.4.1.9.9.388.1.2.2.1.1
           name: cvsChassisSwitchID
         tag: chassis_switch_id
+
+  # RTT info
+  - MIB: CISCO-RTTMON-MIB
+    table:
+      OID: 1.3.6.1.4.1.9.9.42.1.2.10
+      name: rttMonLatestRttOperTable
+    symbols:
+      - OID: 1.3.6.1.4.1.9.9.42.1.2.10.1.1
+        name: rttMonLatestRttOperCompletionTime
+      - OID: 1.3.6.1.4.1.9.9.42.1.2.10.1.2
+        name: rttMonLatestRttOperSense
+    metric_tags:
+      - column:
+          OID: 1.3.6.1.4.1.9.9.42.1.2.9.1.10
+          name: rttMonCtrlOperState
+        table: rttMonCtrlOperTable
```

Comment:
This is meant to be under `column`. This might break with older versions.

Reply:
Then we should change it for the other ones I guess (e.g https://github.com/DataDog/integrations-core/blob/master/snmp/datadog_checks/snmp/data/profiles/_cisco-generic.yaml#L55)